### ISSUE TITLE
stale docs missing 1.16 release

### ISSUE THREAD

**Issue by jedbrown** [python-cffi/cffi#29]
v1.16.0 was released in September, but the default docs (https://cffi.readthedocs.io/en/stable/) only has up to v1.15.1. The latest (https://cffi.readthedocs.io/en/latest/whatsnew.html or https://cffi.readthedocs.io/en/release-1.16/whatsnew.html) docs have v1.16.0rc1 (don't know if the theme change was deliberate), which has pertinent information about working with python-3.12. None of the docs discuss `pyproject.toml`, which is a source of [great consternation](https://gregoryszorc.com/blog/2023/10/30/my-user-experience-porting-off-setup.py/).

**Comment by mattip** [comment]
Perhaps the readthedocs site is pointing to the old mercurial url instead of here, so the web integration is not working?

**Comment by arigo** [comment]
No, they have been updated.  I triggered a new build manually just now.  The version 1.16.0 now appears at https://cffi.readthedocs.io/en/stable/.  The previous "stable" branch failed to build, FWIW.  I myself have no knowledge of `pyproject.toml` and plan to keep it that way, but maybe @nitzmahone can comment on that.
